oca_dependencies.txt

```diff
@@ -0,0 +1 @@
+
```

Comment:
@JayVora-SerpentCS Thanks for your contribution to the OCA. 
Please follow the guidelines from:
https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md

Please remove this file